includes/Stories_Renderer/Carousel_Renderer.php

```diff
@@ -85,22 +90,42 @@ public function render( array $args = [] ) {
 		ob_start();
 		?>
 		<div class="<?php echo esc_attr( $container_classes ); ?>">
-			<amp-carousel
-				width="1"
-				height="1"
-				layout="intrinsic"
-				type="carousel"
-				role="region"
-				aria-label="<?php esc_attr_e( 'Web Stories', 'web-stories' ); ?>"
-			>
+			<?php
+			if ( ! $this->is_amp_request() ) {
+				?>
+				<div class="web-stories-list__carousel">
+					<?php
+					foreach ( $this->story_posts as $story ) {
+						$this->render_single_story_content();
+						$this->next();
+					}
+					?>
+				</div>
+				<div tabindex="0" aria-label="<?php esc_attr_e( 'Previous', 'web-stories' ); ?>" class="glider-prev">&#8592;</div>
+				<div tabindex="0" aria-label="<?php esc_attr_e( 'Next', 'web-stories' ); ?>" class="glider-next">&#8594;</div>
```

Comment:
I think the arrows here need to be translatable as well so that they work in RTL context.

Maybe like this:

```suggestion
				<div tabindex="0" aria-label="<?php esc_attr_e( 'Previous', 'web-stories' ); ?>" class="glider-prev">
					  <?php
						  /* translators: left arrow */
						  esc_html__( '&#8592;', 'web-stories' );
					  ?>
				</div>
				<div tabindex="0" aria-label="<?php esc_attr_e( 'Next', 'web-stories' ); ?>" class="glider-next">
					  <?php
						  /* translators: right arrow */
						  esc_html__( '&#8594;', 'web-stories' );
					  ?>
				</div>
```

Reply:
There won't be any text for these controls as I mentioned above, now that the base style PR is merged, I'll update this.